packages/vow/src/vow.js

```diff
@@ -129,17 +136,29 @@ export const prepareVowKit = zone => {
       },
       watchNextStep: {
         onFulfilled(value) {
-          const { resolver } = this.facets;
+          const { resolver, watchNextStep } = this.facets;
           const { resolve } = getPromiseKitForResolution(resolver);
           if (resolve) {
             resolve(value);
           }
           this.state.stepStatus = 'fulfilled';
-          this.state.value = value;
+          if (zone.isStorable(value)) {
+            this.state.value = value;
+          } else {
+            watchNextStep.onRejected(
+              assert.error(`Vow fulfillment value is not storable: ${value}`),
+            );
+          }
         },
         onRejected(reason) {
           this.state.stepStatus = 'rejected';
```

Comment:
I don't know how I missed this important sync with the ephemera:
```suggestion
          const { resolver } = this.facets;
          const { reject } = getPromiseKitForResolution(resolver);
          harden(reason);
          if (reject) {
            reject(reason);
          }
          this.state.stepStatus = 'rejected';
```